extensions/Khronos/KHR_draco_mesh_compression/README.md

```diff
@@ -127,6 +123,8 @@ easy enough to change the loader to ignore `accessor` without `bufferView` in gl
 definitely open to change this if there actually are some use cases that require
 loading `accessor` independently. 
 
+For effectiveness, it is also valid to not support the fallback which is provided in the above specification, e.g. not providing the data buffer for uncompressed assets. To do that, a glTF file could just ignored the `bufferViews` of `accessors` belong to a `primitive`. This is valid because in glTF 2.0, `bufferView` is not required in `accessor`, although if it is not present, it will be used with `sparse` field to act as a sparse accessor. In this case, Draco must be considered a required extension and should report error if a loader doens't support it.
```

Comment:
This last paragraph isn't clear to me: earlier in this commit, we've removed Draco from `extensionsRequired`, and claimed that loaders should be able "to load the glTF assets no matter [what]". But here, we seem to be saying it's OK to omit the fallback data, in which case the loader must throw an error if it doesn't support Draco?

Wording I would suggest — if it is consistent with what you mean, of course — would be: 

***
*begin suggestion*

The extension provides compressed alternatives to one or more of a primitive's uncompressed attributes. A loader may choose to use the uncompressed attributes instead — when the extension is not supported, or for other reasons. When using compressed Draco data, the corresponding uncompressed attributes defined by the primitive should be ignored. If additional uncompressed attributes are defined by the primitive, but are _not_ provided by the Draco extension, the loader must proceed to use these additional attributes as usual.

> **Implementation note:** To prevent transmission of redundant data, exporters should generally write compressed Draco data into a separate buffer from the uncompressed fallback, and shared data into a third buffer. Loaders may then optimize to request only the necessary buffers.

*end suggestion*
***

IFF we want to also support a mode in which the Draco extension is required (which I have no preference on), then I think that any attributes defined by the Draco extension should be omitted entirely from `primitive.attributes`, or point to accessors without bufferViews as mentioned here. In this case, the Draco extension must also be included in `extensionsRequired`.